streamparse/cli/common.py

```diff
@@ -230,7 +230,10 @@ def resolve_options(cli_options, env_config, topology_class, topology_name,
 
     # If ackers and executors still aren't set, use number of worker nodes
     if not local_only:
-        storm_options['storm.workers.list'] = get_storm_workers(env_config)
+        if 'storm.workers.list' not in storm_options:
+            storm_options['storm.workers.list'] = get_storm_workers(env_config)
+        else:
+            storm_options['storm.workers.list'] = storm_options['storm.workers.list'].split(",")
```

Comment:
Probably best to check that this is a string before doing this a split, incase someone passes in a list.